### ISSUE TITLE
Clearance Level Inheritance resets page view to top

### ISSUE THREAD

**Issue by zcrt** [minvws/nl-kat-coordination#932]
**Describe the bug**
Clearance Level Inheritance resets page view to top

**To Reproduce**
1. Go to object page
2. Click on clearance level inheritance
3. Get redirected to the top of the page

**Expected behavior**
Scrolled down to CLI at 3.

**OpenKAT version**
1.8rc2

**Comment by underdarknl** [comment]
Lets post to the correct Anchor tag like we do with the katalogus.

**Comment by TwistMeister** [comment]
@zcrt You are right. I misunderstood the problem and assumed it was about assigning clearance levels to objects on the objects list page. I'll reopen this issue and fix it properly the next time :)
